App.js

```diff
@@ -0,0 +1,250 @@
+require("dotenv").config()
+const bodyParser = require("body-parser");
+const express = require("express");
+const mongoose = require("mongoose");
+const swaggerjsdocs = require("swagger-jsdoc");
+const swaggerUi = require("swagger-ui-express")
+const jwt = require("jsonwebtoken")
+const bcrypt = require("bcrypt")
+
+const {DoctorRoute} = require("./Routes/doctor.js")
+const {PatientRoute} = require("./Routes/patient.js")
+
+const Doctors = require("./models/doctorModel");
+const Patients = require("./models/patientModel")
+const Hospital = require("./models/hospitalModel")
+
+// const port = process.env.PORT
+
+
+const app = express();
+app.use(bodyParser.urlencoded({extended: true}));
+mongoose.connect('mongodb://localhost:27017/HospitalManagement')
+
+const db = mongoose.connection
+
+db.on('error', (error) => console.log(error))
+db.once('open', () => console.log("connected to database"))
+
```

Comment:
This is the db connection code and it should be placed inside the db file.

Reply:
Shifted to separate [folder](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/2/files/893ce87fe4fe8a5be2c037597a3a6cfbdffc56ac#r1045073805)